Description: POTFILES.in points to a moved file, this also adds the missing files.
Author: Unit 193 <unit193@ninthfloor.org>
Origin: vendor
Forwarded: no
Last-Update: 2014-04-23

--- xfdashboard-0.1.90.orig/po/POTFILES.in
+++ xfdashboard-0.1.90/po/POTFILES.in
@@ -11,7 +11,7 @@
 src/drop-action.c
 src/enums.c
 src/fill-box-layout.c
-src/image.c
+src/image-content.c
 src/live-window.c
 src/live-workspace.c
 src/main.c
@@ -34,4 +34,19 @@ src/window-tracker.c
 src/window-tracker-window.c
 src/window-tracker-workspace.c
 src/workspace-selector.c
-data/themes/default/xfdashboard.theme.in
+src/actor.c
+src/applications-search-provider.c
+src/focus-manager.c
+src/layoutable.c
+src/outline-effect.c
+src/search-manager.c
+src/search-result-container.c
+src/stylable.c
+src/theme-css.c
+src/theme-layout.c
+src/theme.c
+src/tooltip-action.c
+data/themes/xfdashboard/xfdashboard.theme.in
+data/themes/xfdashboard/xfdashboard.xml
+data/xfdashboard-autostart.desktop.in
+data/xfdashboard.desktop.in

